packages/@aws-cdk/aws-pipes-alpha/test/integ.pipe.ts

```diff
@@ -35,28 +41,65 @@ class TestSource implements ISource {
   }
 }
 
+// When this module is promoted from alpha, TestTarget should
+// be replaced with SqsTarget from @aws-cdk/aws-pipes-targets-alpha
+if (!name.endsWith('-alpha')) {
+  throw new Error('aws-pipes has exited alpha, TestTarget should now be replaced with SqsTarget from @aws-cdk/aws-pipes-targets-alpha');
+}
```

Comment:
Is it necessary to throw two errors here with slightly different messages? I think in practice, only the first one would execute, so condensing the message to replace both `TestSource` and `TestTarget` with their SQS counterparts should suffice.